concordium-consensus/src/Concordium/KonsensusV1/LeaderElection.hs

```diff
@@ -147,3 +156,29 @@ updateSeedStateForEpoch newBakers epochDuration ss =
         }
   where
     newNonce = nonceForNewEpoch newBakers ss
+
+-- | Compute the missed rounds for each validator. Starts from the given parent
+--   block up to (not including) the given final round.
+computeMissedRounds ::
```

Comment:
Please add some unit tests.

Reply:
Added a few in `LeaderElectionTests.hs`.